db/models.go

```diff
@@ -16,14 +16,15 @@ type UserConfig struct {
 }
 
 type App struct {
-	ID          uint
-	Name        string `validate:"required"`
-	Description string
-	NostrPubkey string `validate:"required"`
-	CreatedAt   time.Time
-	UpdatedAt   time.Time
-	Isolated    bool
-	Metadata    datatypes.JSON
+	ID                uint
+	Name              string `validate:"required"`
+	Description       string
+	NostrPubkey       string `validate:"required"`
+	WalletChildPubkey string
```

Comment:
I'm wondering about the naming. Is `Child` needed or is it just an implementation detail?

I think actually it would be nice to rename the original `NostrPubkey` too, so we clearly differentiate the two. e.g. `AppPubkey` vs `WalletPubkey`

Reply:
Will rename to WalletPubKey.
Renaming NostrPubkey let's maybe do it on a separate PR?

Reply:
Sounds good, thanks. A TODO could be good though so we don't forget.